### PR TITLE
completed css-ecommerce-page exercise

### PR DESCRIPTION
**Pull request by AnthonyUrbina**
Screen Shot 2022-09-01 at 6 15 41 PM

### CODE REVIEW

File: css-ecommerce-page/styles.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+.row {
+  display: flex;
+  padding-left: 80px;
+  padding-right: 80px;
+}
+
+.one-five {
+  flex-basis: 20%;
+  margin: 10px;
+}
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:24]: Make this into a proper `column ` class. The `margin` should be applied to a separate rule for a separate class that you can add in addition to the `column` class, `column` classes should only have `flex-basis` or `width` applied to them.

File: css-ecommerce-page/styles.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  padding-left: 80px;
+  padding-right: 80px;
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:18]: This should be on a separate class, the `row` class should only have `display: flex` applied to it.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AnthonyUrbina** [comment]
okay sweet. fixed both. had to google how to add 2x classes lol. thanks for the help. 
Screen Shot 2022-09-02 at 9 51 52 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by AnthonyUrbina** [comment]
okay, just fixed this

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**